demo/css/main.css

```diff
@@ -34,3 +34,7 @@ body {
 .dialog.is-active {
     display: block !important;
 }
+
+.is-no-scroll {
```

Comment:
I reckon including `vertical` in this class has value and is `has` better than `is` e.g. `.has-no-vertical-scroll`? Plus we'll need to chain it to the element it's being applied too which should be the `<html>` element. 

It actually makes me wonder if we'll need to namespace our state hooks also as not all state hooks will be chained to a namespaced style hook, like in this instance.

```css
html.has-no-vertical-scroll
```

I'll write up a **discussion/question** type issue for this.